kernel/src/transaction.rs

```diff
@@ -74,9 +72,8 @@ pub struct Transaction {
 impl std::fmt::Debug for Transaction {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         f.write_str(&format!(
-            "Transaction {{ read_snapshot version: {}, commit_info: {} }}",
-            self.read_snapshot.version(),
-            self.commit_info.is_some()
+            "Transaction {{ read_snapshot version: {} }}",
```

Comment:
~either here or create a follow up: can we update this to be a proper debug implementation? it's missing lots of `Transaction` fields now~

actually, how about we just modify this to log `commit_info` (the change related to this PR) and make a follow up to actually expand this to log things we care about